about.html

```diff
@@ -0,0 +1,233 @@
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+
+      <!-- Hamburger -->
+      <div class="hamburger hidden">
+        <div class="hamburger__close-container">
+          <img
+            src="./assets/cancelHamburger.png"
+            alt="close hamburger"
+            class="hamburger__close-img"
+          />
+        </div>
+
+        <ul class="hamburger__show-container">
+          <li>
+            <a href="./index.html" class="menu-links">Home</a>
+          </li>
+          <li><a href="./index.html#program" class="menu-links">Program</a></li>
+          <li><a href="./index.html#join" class="menu-links">Join</a></li>
+          <li>
+            <a href="./index.html#speakers" class="menu-links">Speakers</a>
+          </li>
+          <li><a href="#sponsor" class="menu-links">Sponsor</a></li>
+        </ul>
+      </div>
+
+      <div class="header-container">
+        <!-- Header and navbar -->
+        <header class="header-container__nav">
+          <!-- hamburger icon for mobile -->
+          <div class="hamburger-container">
+            <img
+              src="./assets/hamburgerImg.svg"
+              alt="hamburger Image"
+              class="hamburger-container__img"
+            />
+          </div>
+
+          <!-- desktop -->
+          <div class="desktop-navbar">
+            <div class="header-container__img-text">
+              <a href="index.html" class="header-container__img">
+                <img
+                  src="./assets/logo.svg"
+                  alt=""
+                  class="header-container__logo"
+                />
+              </a>
+              <h4 class="header-container__text">
+                <span class="header-container__text-big"
+                  >IWES INTERNATIONAL</span
+                >
+                <br />
+                <span class="header-container__text-small">
+                  CONFERENCE 2023</span
+                >
+              </h4>
+            </div>
+
+            <nav class="navbar">
+              <ul class="navbar__list">
+                <li>
+                  <a href="./index.html">Home</a>
+                </li>
+                <li>
+                  <a class="active" href="./about.html">About</a>
+                </li>
+                <li>
+                  <a href="./index.html#program">Program</a>
+                </li>
+                <li>
+                  <a href="./index.html#sponsor">Sponsor</a>
+                </li>
+              </ul>
+
+              <h4 class="header-container__campaign">IWES Campaign</h4>
+            </nav>
+          </div>
+        </header>
+
+        <section class="info">
+          <div class="info__box">
+            <h4 class="info__greeting">“Hello! Ladies”</h4>
+            <h1 class="info__meeting-info">
+              INTERNATIONAL WOMEN <br />
+              EMPOWERMENT SCHEME 2023
+            </h1>
```

Comment:
- **_[OPTIONAL]_** Please kindly increase the font size of the h1 on the `About` page also, a larger font size can make the title more prominent and easier to read, which can help draw the user's attention and enhance the user experience. 💪

- Your design
![image](https://user-images.githubusercontent.com/78251927/225397281-525167e3-8129-439c-9023-c4bd12dfaf8e.png)

- Template
![image](https://user-images.githubusercontent.com/78251927/225397676-1e1d3c21-3c97-413f-8f1c-e261a2b1fdde.png)